All EIPIP Meetings/Meeting 011.md

```diff
@@ -0,0 +1,552 @@
+
+**James Hancock**
+I like having the flow that way. Part of the what we've run into historically is the definition of Accepted is kind of clear in the EIP 1. But the community understanding and even sometimes the core developer understanding of what that means is totally different. So having you go through the process, and at any point, it can be rejected is a good way of kind of managing those expectations, and we should show that visually, when we're defining the process. I like that quite a bit. I'll work on doing a document or a PR for the statuses to EIP 1. You're right that it's time to start more defining the network upgrade process. I would just feel more comfortable if we had some more resolution on the standardization process before I start shifting mental gears into that. That's something I can work on.
+
+**Edson Ayllon**
+We could just start opening up a PR to EIP 1. There's nothing stopping us from making future PRs. Yep.
+
+**James Hancock**
+Yeah, I've been hesitant to dump into the network process. I'd just like to get this standardization process at least more moving forward before I start diverting efforts. But I'm open to feedback from the group on what's the best way to proceed there.
+
+**Pooja Ranjan**
+One of the action items mentioned, or not an action item - it's just a part of the decision is like that there should be a particular place to collect all these kind of definitions that we are planning. There is already a mandatory terminology section for EIP. So we are discussing for putting these statuses as described in EIP. It is something that we should consider for EIPs in general as well. If something new is coming up, we should add this section in an EIP. I'm talking about an EIP template that we create.
+
+**Edson Ayllon**
+Yeah, I think we mentioned this in the previous meeting. I had suggested that different standard organizations, they have a section for terminology. It might be good if we simplify to something like, say, definitions as having it mandatory. There may be EIPs that don't really introduce new terms. And then having global terms that are required for each EIP, I don't think is necessary. But some  EIPs may introduce new terminology that would facilitate clarifying to the reader. It brings some value having a definition section.
+
+**James Hancock**
+Yeah, I would agree.
+
+**Pooja Ranjan**
+I'm not too sure how much value it would add if it is not something that is going to be commented. And in most of the EIPs, we can probably go ahead and explain that term in one of the summary sections or somewhere else in the EIP itself. But adding this as a mandatory section for every EIP that is something I'm not sure about, but it doesn't hurt to have something additional as well.
```

Comment:
I'm not too sure how much value it would add if it is not something that is going to be common. And in most of the EIPs, we can probably go ahead and explain that term in one of the summary sections or somewhere else in the EIP itself. But adding this as a mandatory section for every EIP that is something I'm not sure about, but it doesn't hurt to have something additional as well.